timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -37,6 +46,11 @@ def mape_per_item(*, y_true: pd.Series, y_pred: pd.Series) -> pd.Series:
     return ((y_true - y_pred) / y_true).abs().groupby(level=ITEMID, sort=False).mean()
 
 
+def rmsse_per_item(*, y_true: pd.Series, y_pred: pd.Series, random_walk_error: pd.Series) -> pd.Series:
+    mse = mse_per_item(y_true=y_true, y_pred=y_pred)
+    return np.sqrt(mse / random_walk_error)
```

Comment:
For `RMSE` we apply `sqrt` after averaging over items to be compatible with `GluonTS`. I'm not sure if this is the best option, but probably we would want to be consistent between RMSE and RMSSE, so we either need to modify our RMSE definition (+ respective test that checks parity with GluonTS), or perform sqrt after averaging in RMSSE.

Reply:
This is an interesting point. According to M5 competition, the aggregated MRSSE is a weighted average of the item ones:

> After estimating the RMSSE for all the 42,840 time series of the competition, the participating methods will be ranked using the Weighted RMSSE (WRMSSE), as described latter in this Guide, using the following formula:
> WRMSSE=∑_(i=1)^42,840▒〖w_i*RMSSE〗,
> where w_i is the weight of the i_th series of the competition. A lower WRMSSE score is better.

Reply:
I also feel like sqrt before averaging is the more reasonable thing to do, but I would then update the RMSE to do the same.

Reply:
I updated RMSSE to follow the current paradigm.